README.md

```diff
@@ -130,6 +132,16 @@ nnoremap <leader>fb <cmd>lua require('telescope.builtin').buffers()<cr>
 nnoremap <leader>fh <cmd>lua require('telescope.builtin').help_tags()<cr>
 ```
 
+Using Lua:
+
+```lua
+local builtin = require('telescope.builtin')
+vim.keymap.set('n', 'ff', builtin.find_files, {})
```

Comment:
mappings dont have parity with the viml because of missing `noremap = true`

Reply:
@Conni2461 thank you for the comment, however, I don't think `noremap` is a valid option in the context of `vim.keymap.set`. There is an option called `remap` with `false` as the default value which I think is equivalent to `noremap = true` in the context of `nvim_set_keymap`


<img width="595" alt="image" src="https://user-images.githubusercontent.com/2049686/192680088-6e23752d-bce9-422a-88ad-2d33545680e1.png">

Reply:
![image](https://user-images.githubusercontent.com/15233006/193323868-d53ac025-bef9-4eb6-a91c-35bfdf7d16ab.png)

It clearly links to `map-arguments`, noremap should be a map argument.  But remap clearly meantions that the default is already false. maybe the neovim team changed the default to the better.